src/configure/mod.rs

```diff
@@ -145,19 +146,24 @@ pub fn JobConfigureImap(context: &Context) {
             // Step 2: Autoconfig
             4 => {
                 progress!(context, 200);
-                if param.mail_server.is_empty()
+
+                if let Some(new_param) = get_offline_autoconfig(context, &param) {
+                    keep_flags = new_param.server_flags & DC_LP_AUTH_OAUTH2;
```

Comment:
this has to be `keep_flags = param.server_flags & DC_LP_AUTH_OAUTH2;` - we want to preserve the user decision about oauth2